website/utils/plugins.js

```diff
@@ -0,0 +1,41 @@
+const fs = require("fs");
+const path = require("path");
+const matter = require("gray-matter");
```

Comment:
I used `gray-matter` and `title` as those are used by Nextra so we have the same logic